src/main/java/com/xebialabs/overthere/gcp/credentials/ServiceAccountTokenGcpCredentialFactory.java

```diff
@@ -0,0 +1,59 @@
+package com.xebialabs.overthere.gcp.credentials;
+
+import com.google.api.client.auth.oauth2.BearerToken;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.auth.oauth2.TokenResponse;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.compute.ComputeScopes;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Returns credentials defined by a Service Account key in JSON format from the Google Developers Console.
+ */
+class ServiceAccountTokenGcpCredentialFactory extends GcpCredentialFactory {
+
+    private final String projectId;
+    private final String apiToken;
+    private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+    private static final List<String> SCOPES = Arrays.asList(ComputeScopes.CLOUD_PLATFORM);
```

Comment:
It seems that this constant is not used anywhere can you remove it?

Reply:
Will Remove the Constants which are not required :)